go.mod

```diff
@@ -1,15 +1,24 @@
 module github.com/instapro/replikator-api
 
-go 1.12
+go 1.21
```

Comment:
Does this match the go version installed on replikator?

Reply:
I've install 1.23 on all replikators